tiatoolbox/tools/pyramid.py

```diff
@@ -386,15 +394,14 @@ class ZoomifyGenerator(TilePyramidGenerator):
             The WSI reader object. Must implement
             `tiatoolbox.wsicore.wsi_Reader.WSIReader.read_rect`.
         tile_size (int):
```

Comment:
are these variables initialised? may be using super()?

Reply:
Yes, they are initialised in the parent class init (the init is inherited unmodified).